packages/prisma-client/migrations-cli/README.md

```diff
@@ -0,0 +1,50 @@
+## CLI Reference
+
+### `--dev`
+
+This flag can be added to any command. It informs the CLI that it’s used in a development environment, which slightly changes its behaviour:
+
+- automatically loads variables from `.env` files,
```

Comment:
instead of this, cli should just assume the .env file was already read and all variables are in process.env

Reply:
I tried that. But didn't find an easy way to set this up.

Reply:
lets chat

Reply:
Decided to keep as is in chat